projects/storefrontstyles/scss/components/cart/_cart-coupon.scss

```diff
@@ -1,68 +1,65 @@
 @import '../../theme';
 
 %cx-cart-coupon {
-  .couponbox-is-active {
-    .cx-customer-coupons {
-      display: block;
-    }
-  }
   padding: 2rem 3rem;
 
   @include media-breakpoint-down(md) {
     padding: 0 3rem 2rem 3rem;
   }
 
-  .cx-cart-coupon-title {
-    @include type('5');
-    margin: 1.125rem 0 0.5rem;
-  }
-
   .form-group .col-md-4 {
     @include media-breakpoint-up(md) {
       padding: 0;
     }
   }
-  .cx-apply-voucher {
-    position: relative;
-  }
 
-  .cx-customer-coupons {
-    display: none;
-    z-index: 100;
-    position: absolute;
-    max-height: 130px;
-    overflow: auto;
-    width: 100%;
-    background-color: var(--cx-color-inverse);
-    @include type();
+  .cx-cart-coupon-title {
+    @include type('5');
+    margin: 1.125rem 0 0.5rem;
```

Comment:
We probably want to use `var()` here too. As seen in the below code.

Reply:
fix